ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/Ledger.hs

```diff
@@ -316,7 +316,7 @@ instance Crypto c => ValidateEnvelope (ShelleyBlock c) where
   type OtherHeaderEnvelopeError (ShelleyBlock c) =
     STS.PredicateFailure (STS.CHAIN c)
 
-  validateEnvelope cfg ledgerView oldTip hdr = do
+  validateEnvelope cfg (TickedLedgerState _ ledgerView) oldTip hdr = do
```

Comment:
This is somewhat confusing. At least from the names here, this suggests we're taking the full ledger state, which, ticked or not, would allow invalid things to happen here. Surely the ledger view is the correct concept to pass?

Reply:
Yeah, fair enough. WIll rename.